.github/actions/mamba-env/action.yml

```diff
@@ -42,3 +42,7 @@ runs:
         # we set use-only-tar-bz2 to false in order for conda to properly find new packages to be installed
         # for more info see https://github.com/conda-incubator/setup-miniconda/issues/264
         use-only-tar-bz2: false
+    - shell: bash -el {0}
```

Comment:
Why do we need -e flag? It looks like we only use -l everywhere.

Reply:
I've copied it from [`setup-miniconda` docs](https://github.com/marketplace/actions/setup-miniconda#example-1-basic-usage). Removed -e, let's see if it works